### DISCUSSION TITLE
可选择是否去掉这个【 收集/微信】标签

### DISCUSSION THREAD

**Discussion by cjterrence** [Quorafind/Obsidian-Thino#586] [Polls 投票]
### Feature Requested | 请求的功能

因为不需要区分微信或其他方式输入，所以，希望这个默认标签为可选项

### Relevant Screenshot | 相关截图

![CleanShot 2024-04-25 at 14 59 09@2x](https://github.com/Quorafind/Obsidian-Thino/assets/63612976/b06d1ec8-2db8-4236-bdf9-a5e4cfa02305)

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
自定义标签的功能，可能会优先在 thino web 中进行考虑，自定义功能会引入很多不可控的因素。
